code/core/src/storybook-error.ts

```diff
@@ -22,20 +24,19 @@ export abstract class StorybookError extends Error {
 
   /**
    * Specifies the documentation for the error.
-   * - If `true`, links to a documentation page on the Storybook website (make sure it exists before enabling).
+   * - If `true`, links to a documentation page on the Storybook website (make sure it exists before enabling) – This is not implemented yet.
    * - If a string, uses the provided URL for documentation (external or FAQ links).
    * - If `false` (default), no documentation link is added.
    */
-  public documentation: boolean | string | string[] = false;
+  public documentation: boolean | string | string[];
```

Comment:
Should this also be `readonly` like the others?

Reply:
done